cmake/ctest/drivers/rocketman/cron_driver.sh

```diff
@@ -35,21 +35,20 @@ export TDD_HTTPS_PROXY=$https_proxy
 
 . ~/.bashrc
 
+SCRIPT_DIR=`cd "\`dirname \"$0\"\`";pwd`
+export MODULEPATH=$SCRIPT_DIR:$MODULEPATH
+# Trilinos source repo
+export TRILINOS_SOURCE=$SCRIPT_DIR/../../../..
+
+
 # If you update the list of modules, go to ~/code/trilinos-test/trilinos/ and
 # do "git pull". Otherwise, the tests could fail on the first night, as we
 # would first run old cron_driver.sh and only then pull
 
 # ===========================================================================
 export CTEST_CONFIGURATION="default"
-module load sems-env
-module load sems-git
-module load sems-cmake/3.17.1
-module load sems-gcc/5.3.0
-module load sems-openmpi/1.10.1
-module load sems-superlu/4.3/base
-module load sems-hdf5/1.8.12/parallel
-module load sems-netcdf/4.4.1/exo_parallel
-module load sems-boost/1.63.0/base
+module load muelu-gcc
```

Comment:
Do I need to create a custom module for `muelu-gcc`?

Reply:
I added the module file